.changeset/cold-cougars-rest.md

```diff
@@ -0,0 +1,32 @@
+---
```

Comment:
Why do we have 2 changelogs?

Reply:
One reflects upgrade in all components to `^` version of the package, the other one reflects addition of `react-intl` wherever needed. I thought that was a common pattern to differentiate changes in changesets 🤔

Reply:
I see. Well in this case it's fine to have 1 changeset, as it's the same context for the changes.

Multiple changesets are useful to differentiate unrelated changes in different packages.